src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Email addins released
+category: Release Notes
+author: jericho
+---
+
+There are many scenarios where you need to send notifications during your build process, such as notifying colleagues that a build has passed/failed, notifying stakeholders that a package has been published to nuget, etc. One of the most popular ways to send such notification is, of course, email. That's why we recently released three new addins to send emails.
+
+__Cake.Email__
```

Comment:
Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?